assets/js/vue/utils/template-switch.js

```diff
@@ -20,7 +20,7 @@ const templateVariables = {
     tagsExpanded: true,
   },
   GRIIDC: {
-    typeExpanded: false,
+    typeExpanded: true,
```

Comment:
This can also be removed, as the default is true.

Reply:
Okay